features/test/upgrade.feature

```diff
@@ -1,22 +1,58 @@
 Feature: basic verification for upgrade testing
-
-  # note that upuser1 and 2 need to be defined for the environement
+  # @author geliu@redhat.com
   @upgrade-prepare
   @users=upuser1,upuser2
-  Scenario: prepare 2 users
-    Given the user has all owned resources cleaned
+  # @case_id OCP-10017000
+  Scenario: upgrade-prepre cakephp example works well after migrate
     When I run the :new_project client command with:
-      | project_name | upgrade-project-1 |
+      | project_name | project-ocp10017 |
+    Then the step should succeed
+    When I run the :get client command with:
+      | resource | project |
     Then the step should succeed
+    And the output should contain:
+      | project-ocp10017 |
+  
+  #@author geliu@redhat.com
+  #@users=upuser1,upuser2
+  #@case_id OCP-10017
+  @admin
+  Scenario: cakephp example works well after migrate
+    Given I switch to cluster admin pseudo user
+    Given I use the "project-ocp10017" project
```

Comment:
@geliu2016 is having the project enough of a check

Reply:
@pruan-rht  aha, this is an example testcase, you may find this case is very old and inactive testcase, I active it for my test, and it is very simple for other team to understand and follow this case format, and also i need 2 automation cases to verify the auto works well for multiple testcase(multiple means number >= 2), sorry for have not add 'example' sign for it, and it will be deleted soon, thx